docker/entrypoint.sh

```diff
@@ -203,7 +203,11 @@ then
     /bin/chmod g+s /usr/sbin/ssmtp
 
     mkdir -p /var/www/cms/library/temp
-    chown apache.apache -R /var/www/cms
```

Comment:
Unfortunately these don't do the same job as before.

The issue is that users may modify files inside those directories themselves, and this step when run ensures that the folders and their contents have correct permissions.

Making each of those recursive would probably do the same job, but will probably be just as slow unfortunately.

Reply:
Understood, I tested the recursion on the mounted volumes only and it seems to be working just as fast as my last commit. (I don't have any custom files in the mounted volumes).

I have commited a new version.